org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/IBreakpoint.java

```diff
@@ -0,0 +1,8 @@
+package org.eclipse.jdt.ls.debug;
+
+import com.sun.jdi.request.EventRequest;
+
+public interface IBreakpoint {
+    // This is the receipt for tracking the breakpoint at JDI side
```

Comment:
For the comment on the public API, it is better to use the javadoc style. /** .. */